package.json

```diff
@@ -2,7 +2,7 @@
   "name": "qtip2",
   "title": "qTip2 - Pretty powerful tooltips",
   "description": "Introducing... qTip2. The second generation of the advanced qTip plugin for the ever popular jQuery framework.",
-  "version": "3.0.3",
+  "version": "3.0.4",
```

Comment:
not likely they will ever release 3.0.4 but if for some reason 
the versions will be confusing 
If not an issue can we change to 3.0.4-sirenpath1
or something like that

Reply:
Changed